cdc/owner_test.go

```diff
@@ -22,6 +22,11 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+
+	"github.com/pingcap/tidb/store/tikv/oracle"
+
```

Comment:
Please optimize the imports.

Reply:
ok

Reply:
The package `github.com/pingcap/tidb/store/tikv/oracle`  needs to be imported. Otherwise I have to write a method to get the physical time from the TS.

Reply:
We should avoid empty lines among the imports, unless it's the one separating standard lib imports and custom/third-party imports.

Reply:
ok, I get it.